.gitignore

```diff
@@ -8,3 +8,4 @@ test.ts
 qdrant_storage
 .*.swp
 .rgignore
+core-api-keys.json
```

Comment:
note: this will require having a file like this one somewhere, and having the path to it in you local core secrets (as `API_KEYS`)

Reply:
Not a big fan of that as it complexity the dev setup for interviews

We could fail open in the absence of file but that's not desirable from a security perspective...

Why not use a simple env var?

Reply:
Mostly because it's annoying to set using kube CLI due to escaping. 

I can also add an env var to disable check that we can set in dev, will be just another line in the env file, wdyt ?

Reply:
Yes that seems reasonable